test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -207,18 +228,26 @@ test_ucp_tag::send(entity &sender, send_type_t type, const void *buffer,
         }
         break;
     case SEND_NBR:
-        req = request_alloc();
-        status = ucp_tag_send_nbr(sender.ep(worker_index), buffer, count,
-                                  datatype, tag, req);
-        ASSERT_UCS_OK_OR_INPROGRESS(status);
-        if (status == UCS_OK) {
-            request_free(req);
+        param.op_attr_mask |= UCP_OP_ATTR_FIELD_REQUEST;
```

Comment:
will you add more tests in next PRs? Like external request with callback, or UCX request without callback, etc?